### PR TITLE
UCP/TEST: added smoke tests for mlane

### PR DESCRIPTION
**Pull request by hoopoepg**
- added smoke tests for multi-lane transfer

### CODE REVIEW

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/6106/ for details (Mellanox internal link).

**Comment by hoopoepg** [comment]
bot:mlx:retest

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/6108/ for details (Mellanox internal link).

**Comment by hoopoepg** [comment]
bot:mlx:retest

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/6110/ for details (Mellanox internal link).

File: test/gtest/ucp/test_ucp_tag_xfer.cc [hunk 1]
```diff
@@ -986,6 +986,36 @@ UCS_TEST_P(test_ucp_tag_stats, rndv_unexpected, "RNDV_THRESH=1000",
                       UCP_WORKER_STAT_TAG_RX_RNDV_UNEXP);
 }
 
+UCS_TEST_P(test_ucp_tag_xfer, generic_exp_mlane_eager,
+           "MAX_EAGER_LANES=3") {
+    test_xfer(&test_ucp_tag_xfer::test_xfer_generic, true, false, false);
+}
+
+UCS_TEST_P(test_ucp_tag_xfer, generic_exp_mlane_rndv,
```
Comment (yosefe) [inline test/gtest/ucp/test_ucp_tag_xfer.cc]: why just generic? maybe add some tests for contig, or just make all tests in this test suite use MAX_EAGER_LANES=2,MAX_RNDV_LANES=2? it's possible to set it from test::init() function

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/6123/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/6124/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/6126/ for details (Mellanox internal link).

**Pull request merged**